README.md

```diff
@@ -1,181 +1,53 @@
-# Tracee
-Container and system tracing using eBPF
+# Tracee - Container and system tracing using eBPF
 
-**Tracee** is a lightweight, easy to use container and system tracing tool.
-After launching the tool, it will start collecting traces of newly created containers (container mode) or processes (system mode).
-The collected traces are mostly system calls performed by the processes,
-but other events, such as capabilities required to perform the actions requested by the process, are also supported.
+Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real time.
```

Comment:
I think it should be noted (as already was) that it only collet traces of newly created processes or containers (you will not see events of processes which already executed before Tracee was started)

Reply:
https://github.com/aquasecurity/tracee/pull/70/commits/5c88c1c03d3ed3e56141261abfa7bad39e6fe5b8